example/mbed-os-5/mbed_app.json

```diff
@@ -0,0 +1,15 @@
+{
+    "macros": [
+        "MEM_ALLOC=malloc",
+        "MEM_FREE=free"
+    ],
+    "target_overrides": {
+        "*": {
+            "platform.stdio-baud-rate": 115200,
+            "platform.stdio-convert-newlines": true,
+            "platform.stdio-buffered-serial": true,
+            "target.features_add": ["COMMON_PAL"],
```

Comment:
Btw, this is no longer necessary. The FEATURE_COMMON_PAL is now available by default \o/.